Cabal/Distribution/Types/PackageName.hs

```diff
@@ -22,7 +22,7 @@ import Distribution.Parsec
 --
 -- @since 2.0.0.2
 newtype PackageName = PackageName ShortText
-    deriving (Generic, Read, Show, Eq, Ord, Typeable, Data)
+    deriving (Generic, Read, Show, Eq, Ord, Typeable, Data, NFData)
```

Comment:
Unnecessary change.

I'd rather not do it, as we have to support `deepseq` versio nspan where Generic derivation does different things. Therefore, I'd rather see all `NFData` instances be written explicitly.

Reply:
I've dropped the commit that introduced the change.